pinot-server/src/main/java/org/apache/pinot/server/starter/ServerInstance.java

```diff
@@ -125,9 +125,14 @@ public ServerInstance(ServerConf serverConf, HelixManager helixManager, AccessCo
 
     if (serverConf.isMultiStageServerEnabled()) {
       LOGGER.info("Initializing Multi-stage query engine");
+      if (serverConf.isMultiStageTlsServerEnabled()) {
+        // TODO: Dispatch clients cannot be configured to use TLS, so using TLS for multi-stage query engine is not
```

Comment:
```suggestion
        // TODO: Dispatch clients currently can't be configured to use TLS, so using TLS for multi-stage query engine is not
```
nit: the existing wording makes it sound like there's some technical limitation rather than it just being an implementation TODO